abuseipdb/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM komand/python-3-37-slim-plugin:3
+FROM rapid7/insightconnect-python-3-38-plugin:4
```

Comment:
I think Mike caught this on the other one.  Are we able to keep using slim here?